lua/lvim/core/which-key.lua

```diff
@@ -82,7 +82,7 @@ M.config = function()
       [";"] = { "<cmd>Alpha<CR>", "Dashboard" },
       ["w"] = { "<cmd>w!<CR>", "Save" },
       ["q"] = { "<cmd>lua require('lvim.utils.functions').smart_quit()<CR>", "Quit" },
-      ["/"] = { "<Plug>(comment_toggle_linewise_current)", "Comment toggle current line" },
+      ["/"] = { "<Plug>(comment_toggle_current_linewise)", "Comment toggle current line" },
```

Comment:
this is the _deprecated_ function between the two, did you run `:PackerSync` ?

Reply:
Ah, my bad. I misunderstood. It works properly after `:PackerSync`. Sorry!